tfhe/src/core_crypto/commons/numeric/mod.rs

```diff
@@ -40,6 +40,9 @@ pub trait Numeric:
     const MAX: Self;
 }
 
+pub trait UnsignedNumeric: Numeric {}
+pub trait SignedNumeric: Numeric {}
+
```

Comment:
wasn't there a sort of "mini" Integer trait somewhere ? We may want to revisit those a bit

Reply:
There is a MiniInteger trait defined and used for scalar division, but its not worth to make it public has it only contains bounds of traits used by scalar division so probably too limited for other cases / too restricting for others)

Here the purpose of SignedNumeric, UnsignedNumeric is not the same Unsigned/SignedInteger (there is no bounds to any std::ops traits), its just used as marker to distinguish between Signed and Unsigned without having a trait like Unsigned/SignedInteger as it would be too restrictive